.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,129 @@
+name: 🐞 Bug Report
+description: File a bug report
+title: '[Bug]: '
+labels: ['Type: Bug 🐛', 'Needs: Triage 🔍']
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+  - type: dropdown
+    id: lib-version
+    attributes:
+      label: Library
+      description: Which library is affected
+      options:
+        - React Northstart / v0 (@fluentui/react-northstart)
+        - React / v8 (@fluentui/react)
+        - React Components / vNext (@fluentui/react-components)
+        - Web Components (@fluentui/web-components)
+    validations:
+      required: true
+
+  - type: textarea
+    id: env-info
+    attributes:
+      label: System Info
+      description: Output of `npx envinfo --system --npmPackages '{@fluent/*}' --browsers`
+      render: shell
+    validations:
+      required: true
+
+  - type: dropdown
+    id: a11y-bug
+    attributes:
+      label: Are you reporting Accessibility issue?
+      description: |
+        **PLEASE NOTE:**
+
+        1. Do not link to, screenshot or reference a Microsoft product in this description.
+        2. Our screen reader support is limited to Edge + Narrator.
```

Comment:
I checked briefly with @jurokapsiar and it's not true for Northstar.

I don't know exact requirements, but it's probably not true for v9, too (manual accessibility reviews include screen readers, for example https://github.com/microsoft/fluentui/issues/22345).

Reply:
Makes sense to me 👍